report/src/App.tsx

```diff
@@ -14,6 +14,12 @@ import { z } from "zod";
 const RED_THRESHOLD = 20;
 const GREEN_THRESHOLD = 80;
 
+const versionSubtitles = {
+  workerd: "node_compat = true",
```

Comment:
```suggestion
  workerd: "nodejs_compat",
```

Reply:
Good call on adding the activation flags here.

Except, you had the flags a bit wrong wrong, so I suggested fixes to match what each column really represents.

Reply:
Ah whoops! All the more reason to add them for others :)